### PR TITLE
11.0 mig product brand

### PR DESCRIPTION
**Pull request by Daniel-CA**
This supersedes #301

### CODE REVIEW

**Review by anajuaristi — APPROVED** [review]
:+1: 
tested

**Comment by pedrobaeza** [comment]
Please squash commits by logical set/author, as there are a lot of them that are pure minor corrections (`Update author name` or similar), so this way we clean a bit the commit history now that we have the occasion.

File: product_brand/models/product_brand.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        'product_brand_id',
+        string='Brand Products',
+    )
+    products_count = fields.Integer(
+        string='Number of products',
+        compute='_get_products_count',
+    )
+
+    @api.one
+    @api.depends('product_ids')
```
Comment (JayVora-SerpentCS) [inline product_brand/models/product_brand.py]: You may avoid using api.one

**Comment by Daniel-CA** [comment]
Commits squashed and code improved.

**Comment by JayVora-SerpentCS** [comment]
(y)

**Comment by Daniel-CA** [comment]
Can this one be merged?

**Comment by Daniel-CA** [comment]
@pedrobaeza  @guewen @yvaucher @sebastienbeau Pinging product-attribute commiters.

**Comment by pedrobaeza** [comment]
@Daniel-CA if you want reviewers, you can review other PRs in this repository, and ask them to review yours in return.

**Comment by Daniel-CA** [comment]
Thanks @pedrobaeza,  I usually do it this way, anyway this PR already has 2 reviews from @anajuaristi  and @JayVora-SerpentCS, don't know how many more are needed or if it has to be from an OCA commiter.

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
We may proceed guys!

**Pull request merged**